src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,44 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/createCar")
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("mate");
+    private static final CarService carService
+            = (CarService) injector.getInstance(CarService.class);
+    private static final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
```suggestion
    public void doGet(HttpServletRequest req, HttpServletResponse resp)
```
in controllers u have this methods or public, or protected. choose one option)(check other controllers and their  access modifiers)